msrcrypto.js

```diff
@@ -15,7 +15,9 @@
 //    limitations under the License.
 //
 //*******************************************************************************
-if (typeof module !== 'undefined') {
+var nodeJsSupport = (typeof module !== 'undefined') && (this.module !== module)

```

Comment:
sadly, no. without these changes in msrCrypto.js, that file will not run under node ... i haven't heard back form the MSR folks, but hope they will respond soon...

all i can say is that, with this PR, the smallest number of changes necessary are present...